src/main/java/sample/views/FreePlayWindow.java

```diff
@@ -104,11 +119,40 @@ private void initialize() {
         }
     }
 
+
+    /**
+     * This function assigns the text-note name on the button that is pressed.
+     * On white keys it's simple because they are big. On the black keys the text displays
+     * vertically so that the user can view the whole note name. It is called via show_notes action
+     * listener
+     */
+    private void Display_Notes_help()
+    {
+        for (Button button : whiteKeys)
+        {
+            String text = button.getTooltip().getText();
+            text = text.substring(0, text.length() -1);
+            button.setText(text);
+            button.setAlignment(Pos.BOTTOM_CENTER);
+        }
+
+        for(Button button : blackKeys)
+        {
+            String text = button.getTooltip().getText();
+            text = text.substring(0, text.length() -1);
+            button.setFont(new Font("aerials", 8));
+            button.setText(text);
+            button.setTextFill(Color.WHITE);
+            button.setAlignment(Pos.BOTTOM_LEFT);
```

Comment:
I would prefer to have this text centered on the black keys. Will create a separate issue for that.